linuxbrew-core/Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM ubuntu
 MAINTAINER Shaun Jackman <sjackman@gmail.com>
 
+ENV DEBIAN_FRONTEND noninteractive
+
 RUN apt-get update
-RUN apt-get install -y curl g++ gawk m4 make patch ruby tcl
+RUN apt-get install -qq curl g++ gawk m4 make patch ruby tcl git
 
 RUN useradd -m linuxbrew
 RUN echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
 
 USER linuxbrew
-WORKDIR /home/linuxbrew
-ENV PATH /home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH
-RUN yes |ruby -e "$(curl -fsSL https://raw.github.com/Homebrew/linuxbrew/go/install)"
+ENV unset LD_LIBRARY_PATH HOMEBREW_CC
+ENV prefix /home/linuxbrew/linuxbrew
+ENV PATH $prefix/bin:$prefix/sbin:$PATH
+
+RUN git clone https://github.com/Homebrew/linuxbrew.git $prefix
```

Comment:
I wanted to extend this to standalone installation and hence the unsetting, prefixing.